simulation/traffic_simulator/include/traffic_simulator/traffic_lights/configurable_rate_updater.hpp

```diff
@@ -0,0 +1,50 @@
+// Copyright 2015 TIER IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TRAFFIC_SIMULATOR__TRAFFIC_LIGHTS__CONFIGURABLE_RATE_UPDATER_HPP
+#define TRAFFIC_SIMULATOR__TRAFFIC_LIGHTS__CONFIGURABLE_RATE_UPDATER_HPP
+
+#include <rclcpp/rclcpp.hpp>
+#include <traffic_simulator/traffic_lights/traffic_light_manager.hpp>
+
+namespace traffic_simulator
+{
+class ConfigurableRateUpdater
```

Comment:
I have the impression that `ConfigurableRateUpdater` is a class that updates the rate.
How about `RateConfigurableMessagePublisher` ?

Reply:
I think the class itself does not publish messages, so putting `publisher` into the name could be confusing. In my opinion, the current name expresses the class functionality quite well. 

If requested, I can change the name of the class.

Reply:
OK.
It is a small problem.
So, I close this conversation.